flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -75,6 +76,7 @@
     private static final String SELECT_NOTIFICATION = "SELECT_NOTIFICATION";
     private static final String SCHEDULED_NOTIFICATIONS = "scheduled_notifications";
     private static final String INITIALIZE_METHOD = "initialize";
+    private static final String CREATE_CHANNEL_METHOD = "create_notification_channel";
```

Comment:
the name of the constant doesn't match existing convention, which would have resulted in this being named `CREATE_NOTIFICATION_CHANNEL_METHOD`